gamepage_resize.py

```diff
@@ -0,0 +1,490 @@
+        if (x + w) * self.scale + image_x > cur[0] > x * self.scale + image_x and\
+                (y + h) * self.scale + image_y > cur[1] > y * self.scale + image_y:
+            if action is not None and action != "home":
+                pg.draw.rect(self.image, ac, (x, y, w, h))
+            if click[0] == 1 and action is not None:
+                if action == "quit":
+                    self.done = True
+                if action == "soundOn":
+                    pass
+                if action == "soundOff":
+                    pass
+                if action == "challenge" and self.client:
+                    self.client.sendChallenge(msg)
+                if action == 'listOnline':
+                    if not self.client:
+                        self.client = startClient()
+                        self.client.pump()
+                if action == "player1":
+                    self.page = OnePlayerOptions()
+                if action == "player2":
+                    self.page = TwoPlayerOptions()
+                if action == "settings":
+                    self.page = Settings()  # loadSettingsPage()
+                if action == "PlayOnline":
+                    print("clicked play online")
+                if action == "LocalGame":
+                    self.page = GamePage(False)
+                if action == "p1":                        # WHAT IS THIS
+                    print("In action p1")
+                if action == "p2":
+                    print("In action p2")
+                # game loop for one player
+                if action == "main1":
+                    self.page = GamePage(True)
+                # game loop for two player
+                if action == "main2":
+                    self.page = GamePage(False)
+                if action == "home":
+                    self.page = Intro()
+        else:
+            if action is not None and action != "home":
+                pg.draw.rect(self.image, ic, (x, y, w, h))
+
+    def settings_quit_btn(self):
+        if not isinstance(self.page, WinPage):
+            self.button("quit", 515, 425, 70, 20, tan_color, tan_highlight, 'quit')
+            self.image.blit(render_text(15, "Quit", black_color), (535, 423))
+        if not isinstance(self.page, Settings) and not isinstance(self.page, WinPage):
+            self.button("settings", 515, 20, 70, 20, tan_color, tan_highlight, 'settings')
+            self.image.blit(render_text(15, "Settings", black_color), (520, 18))
+
+    def load_intro_buttons(self):
+        self.button("player1", 125, 300, 150, 34, tan_color, tan_highlight, 'player1')
+        self.image.blit(render_text(28, "1-Player", black_color), (150, 295))
+        self.button("player2", 325, 300, 150, 34, tan_color, tan_highlight, 'player2')
+        self.image.blit(render_text(28, "2-Player", black_color), (350, 295))
+
+    def load_settings_buttons(self):
```

Comment:
It would make more sense to me for these functions to be in the Settings page class. Any reason you included them here instead?